seisbench/models/base.py

```diff
@@ -213,6 +344,23 @@ def _parse_metadata(self):
         default_args = self._weights_metadata.get("default_args", {})
         self.default_args.update(default_args)
 
+    def _get_input_args(self, obj):
```

Comment:
Could we move this function to `test_models.py` as a helper? I think it's not used anywhere except in the tests.

Reply:
Yeah makes complete sense. This was legacy code from the old approach but is now not required as you say, moved to tests.